parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/ParallelConsumerOptions.java

```diff
@@ -86,19 +87,71 @@ public enum ProcessingOrder {
      */
     public enum CommitMode {
 
+        // tag::transactionalJavadoc[]
         /**
-         * Periodically commits through the Producer using transactions. Slowest of the options, but no duplicates in
-         * Kafka guaranteed (message replay may cause duplicates in external systems which is unavoidable with Kafka).
+         * Periodically commits through the Producer using transactions.
          * <p>
-         * This is separate from using an IDEMPOTENT Producer, which can be used, along with {@link
-         * CommitMode#PERIODIC_CONSUMER_SYNC} or {@link CommitMode#PERIODIC_CONSUMER_ASYNCHRONOUS}.
+         * The benefits of using this mode are:
```

Comment:
Make sure to clearly explain that unlike normal use of the raw clients, parallel consumer will mix together records from different concurrent operations into the single transaction as the producer in science is shared between current work. If you would like support for separate transactions per processing action please file an issue.